src/fu-tool.c

```diff
@@ -1547,6 +1571,10 @@ main (int argc, char *argv[])
 		priv->flags |= FWUPD_INSTALL_FLAG_ALLOW_OLDER;
 	if (force)
 		priv->flags |= FWUPD_INSTALL_FLAG_FORCE;
+	if (priv->internal_only && priv->external_only) {
```

Comment:
smells like a bit of a hack. wouldn't something like `--filter=internal` be better? PackageKit supports `--filter=~internal' for the opposite. Then we could support other things to filter in the future. The flags can be parsed using the existing enums.